README.md

```diff
@@ -84,6 +84,33 @@ implicit val fooOWrites: OWrites[Foo] =
 
 In case you need even more control, you can still implement your own `TypeTagOWrites` and `TypeTagReads`.
 
+### Custom format for certain types in hierarchy
+
+Sometimes, you might want to represent one type differently than default format would. This can be done by creating an instance of `DerivedReads` or `DerivedWrites` for said type. Below is an example of implementing both custom reads and writes for a single class in a hierarchy:
+
+~~~ scala
+sealed trait Hierarchy
+case class First(x: Integer)
+case class Second(y: Integer)
+
+implicit val SecondReads: DerivedReads[Second] = new DerivedReads[Second] {
+  def reads(tagReads: TypeTagReads, adapter: NameAdapter) = (__ \ "foo").read[Integer].map(foo => Second(foo))
```

Comment:
You should also check that the `type` field is equal to `"Second"`, to be consistent with your encoder.

Reply:
Done